pkg/rpcclient/rpc_test.go

```diff
@@ -831,6 +832,42 @@ var rpcClientTestCases = map[string][]rpcClientTestCase{
 			},
 		},
 	},
+	"getproof": {
```

Comment:
Could we also add an integrational test to `pkg/services/rpcsrv/client_test.go`, please? We have a special pre-filled testchain, you can get proof related data from it. Check the way how it is done for RPC server tests at the https://github.com/nspcc-dev/neo-go/blob/de7bff9cea3215697c0ecaf851220cd233f05bad/pkg/services/rpcsrv/server_test.go#L2033

Reply:
Will do.

Reply:
Done.